d2/src/test/java/com/linkedin/d2/xds/TestXdsClientImpl.java

```diff
@@ -251,23 +252,35 @@ public Object[][] provideBadD2URIMapDataTestCases()
   }
 
   @Test(dataProvider = "badD2URIMapUpdateTestCases")
```

Comment:
the var name at line 82 for the empty data case can be DISCOVERY_RESPONSE_URI_MAP_EMPTY_DATA then.

Reply:
Done